components/atoms/LayoutToggle/layout-toggle.tsx

```diff
@@ -16,26 +16,32 @@ const LayoutToggle = ({ value, onChange }: LayoutToggleProps) => {
 
   return (
     <div className="flex h-8 border rounded-lg shadow-xs w-28 p-[1px] text-light-slate-9">
-      <div
+      <button
```

Comment:
This could be an actual radio button, but for the time being with how it's styled, I opted for the `role="radio` with `aria-checked`.